web/eslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```

Comment:
also you still need a jscrc file for now. ass all the rules arent intergrated yet @grahamgilchrist ?